File: 이취코/jeongseon/week3/ch12/럭키스트레이트.cpp
#include <iostream>
using namespace std;

int n, mid, lsum, rsum;
string sn;

int main(){

    cin >> n;

    sn = to_string(n);

    mid = sn.size() / 2;

    //cout << mid;

    for(int i=0; i<mid; i++){
        lsum += sn[i]-'0';
    }
    for(int i=mid; i<sn.size(); i++){
        rsum += sn[i]-'0';                    //for문 하나로 해결 가능
    }

    if(lsum == rsum) cout << "LUCKY";
    else cout << "READY";


}

File: 이취코/Jihwan/week3/ch12/럭키_스트레이트.py
N = input()
middle = len(N) // 2

r1 = 0
r2 = 0
for i in range(middle):
    r1 += int(N[i])
    r2 += int(N[i+middle])

if r1 == r2:
    print('LUCKY')
else:
    print("READY")